godel_surface_detection/include/services/surface_blending_service.h

```diff
@@ -81,7 +87,13 @@ struct ProcessPlanResult
 class SurfaceBlendingService
 {
 public:
-  SurfaceBlendingService();
+  SurfaceBlendingService() : publish_region_point_cloud_(false),
```

Comment:
Unless this needs to be in the header, I would prefer the constructor to be in the cpp file.